backend.go

```diff
@@ -226,7 +227,7 @@ type Backend interface {
 	//
 	// The size can be used if the backend needs to read the whole reader; use
 	// gofakes3.ReadAll() for this job rather than ioutil.ReadAll().
-	PutObject(bucketName, key string, meta map[string]string, input io.Reader, size int64) (PutObjectResult, error)
+	PutObject(bucketName, key string, meta map[string]string, input io.Reader, size int64, storageClass string) (PutObjectResult, error)
```

Comment:
You may use the `StorageClass` type here.

Reply:
done.